src/common/Config.ts

```diff
@@ -14,9 +14,20 @@
  * limitations under the License.
  */
 
-import { isNode } from '../environment.js';
+interface Config {
+  WebSocketTransportClass: any;
+  fetch: any;
+  fs: any;
+  path: any;
```

Comment:
I'm not sure I love defining these as any. 

I think we could do `typeof import('fetch') | null` or something like that?

Reply:
you're right, I should define these as explicit clear types.
I'm not so familiar with typescript, I'll look into it.